src/assets/src/components/common.tsx

```diff
@@ -473,3 +473,45 @@ export const Breadcrumbs = (props: BreadcrumbsProps) => {
 
     );
 }
+
+interface QueueListProps {
+    queues: ReadonlyArray<QueueBase>;
+    manageLink?: boolean | undefined;
+}
+
+export function QueueList (props: QueueListProps) {
+    const linkBase = props.manageLink ? '/manage/' : '/queue/'
+    const badgeClass = 'queue-list-badge'
+
+    const queueItems = props.queues.map(q => (
+        <Link to={`${linkBase}${q.id}`}>
```

Comment:
Adding a `key` to the top-level element will get rid of the warning in the console (`Each child in a list should have a unique "key" prop.`).

Reply:
Hm, must have missed that. Will take a look.

Reply:
Well, there is one on `ListGroup.Item`, but I guess I failed to check for warnings after I moved the `Link` component out. I just moved the `key` up to `Link`; see https://github.com/tl-its-umich-edu/remote-office-hours-queue/pull/153/commits/56641a49b12ae8f1823900f9b6d6c8656a8a2dc9